src/Traits/JsonPathAccessor.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
We should add a comment specifying that this is only partly implemented (only a small subset of jsonpath operations is currently supported)

Reply:
If we need the spec, we can use third parties like
https://github.com/FlowCommunications/JSONPath
or
https://github.com/Galbar/JsonPath-PHP

Reply:
Oh, I don't know how I missed that, but yes, we can substitute it with the suggested package.

Reply:
None work with objects.